sycl/include/CL/sycl/detail/pi.h

```diff
@@ -106,7 +106,7 @@ typedef enum {
   PI_MEM_OBJECT_IMAGE1D        = CL_MEM_OBJECT_IMAGE1D,
   PI_MEM_OBJECT_IMAGE1D_ARRAY  = CL_MEM_OBJECT_IMAGE1D_ARRAY,
   PI_MEM_OBJECT_IMAGE1D_BUFFER = CL_MEM_OBJECT_IMAGE1D_BUFFER
-} _pi_mem_object_type;
+} _pi_mem_type;
```

Comment:
rename PI_MEM_OBJECT_... to PI_MEM_TYPE...

Reply:
Can we resolve it in a separate patch?

Reply:
OK, no problem. But please follow up asap, before people get a chance to use it.